client/human/HumanClientApp.cpp

```diff
@@ -411,7 +411,20 @@ namespace {
 #include <stdlib.h>
 #endif
 
+namespace {
+    class LocalServerAlreadyRunningException : public std::runtime_error {
+    public:
+        explicit LocalServerAlreadyRunningException(const std::string& s) : std::runtime_error(s) {}
```

Comment:
Is this ever created with another string parameter? If not, can remove the parameter and set it in the constructor, rather than needing to pass it each time...